src/server/config/swagger.json

```diff
@@ -49,6 +49,40 @@
         ]
       }
     },
+    "/users/name": {
+      "get": {
+        "tags": ["user name"],
+        "summary": "Get a name for specific user by uid",
+        "description": "",
+        "operationId": "getName",
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "name": "uid",
+            "in": "body",
```

Comment:
change to header, similar to swagger in this PR https://github.com/HackYourFuture-CPH/FP-class11/pull/154